### PR TITLE
Init

### PR DESCRIPTION
**Pull request by goranchik**
initial implementation

### CODE REVIEW

File: src/main/java/net/corevalue/multitenant/poc/config/MultiTenantProperties.java [hunk 1]
```diff
@@ -24,6 +24,7 @@
     @Getter @Setter
     static class OrientDb {
```
Comment (andyislegend) [inline src/main/java/net/corevalue/multitenant/poc/config/MultiTenantProperties.java:25]: Make this @ ConfigurationProperties to enable config through a properties file

File: src/main/java/net/corevalue/multitenant/poc/config/OrientDBConfiguration.java [hunk 2]
```diff
@@ -22,7 +22,8 @@
 
     @Bean
     OrientDB orientDB(MultiTenantProperties props) {
-        return new OrientDB(String.format("remote:%s", props.getOrientDb().getHost()), OrientDBConfig.defaultConfig());
```
Comment (andyislegend) [inline src/main/java/net/corevalue/multitenant/poc/config/OrientDBConfiguration.java]: with ConfigurationProperties use @Value

**Review by andyislegend — CHANGES_REQUESTED** [review]

File: src/main/java/net/corevalue/multitenant/poc/config/MultiTenantProperties.java (hunk 1 above)
Comment (goranchik) [reply src/main/java/net/corevalue/multitenant/poc/config/MultiTenantProperties.java:25]: It's already done, or probably I missed something?

File: src/main/java/net/corevalue/multitenant/poc/config/OrientDBConfiguration.java (hunk 2 above)
Comment (goranchik) [reply src/main/java/net/corevalue/multitenant/poc/config/OrientDBConfiguration.java]: What is the advantage of using the @value approach?

File: src/main/java/net/corevalue/multitenant/poc/config/MultiTenantProperties.java (hunk 1 above)
Comment (andyislegend) [reply src/main/java/net/corevalue/multitenant/poc/config/MultiTenantProperties.java:25]: yes, PR doesn't show full code.

**Review by andyislegend — APPROVED** [review]

**Pull request merged**